lib/stack.js

```diff
@@ -1,24 +1,28 @@
+const LinkedList = require("./linked-list");
+
 class Stack {
     constructor() {
-        // this.store = ...
-        throw new Error("This method has not been implemented!");
+        this.store = new LinkedList();
     }
 
-    push() {
-        throw new Error("This method has not been implemented!");
+    push(value) {
+        this.store.addFirst(value);
     }
 
     pop() {
-        throw new Error("This method has not been implemented!");
+        const first = this.store.getFirst();
+        this.store.delete(first);
+        return first;
     }
 
     isEmpty() {
```

Comment:
👀 This works, however LinkedList also has an `empty` method you could take advantage of